docs/DeveloperGuide.md

```diff
@@ -2,37 +2,254 @@
+the patient might have, and dispense medicine and dosage accordingly, at a speed much faster than waiting at a clinic.
 
 ## User Stories
 
-|Version| As a ... | I want to ... | So that I can ...|
-|--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
+| Version | As a ...                            | I want to ...                                                                                                    | So that I can ...                                                                                      |
+|---------|-------------------------------------|------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------|
+| v1.0    | new user                            | see instructions that are clear and succinct on how to use the app                                               | refer to them when I forget how to use the application, without needing to consult anyone              |
+| v1.0    | new user                            | view medicine dosages prescribed easily                                                                          | take medication safely.                                                                                |
+| v1.0    | new user                            | check what kind of illness I have based on a strange combination of symptoms I woke up with                      | find out if I need to consult a doctor and prevent myself from wasting time queueing to see one.       |
+| v1.0    | office worker feeling a bit ill     | quickly get medicine during lunch break                                                                          | get back to work quickly                                                                               |
+| v1.0    | patient who dislikes seeing doctors | be prescribed medication without consulting a doctor                                                             | avoid interacting with doctors.                                                                        |
+| v1.0    | patient                             | to know whether my symptoms are serious enough to warrant a trip to the doctor or would some medicine be enough. | save time on visiting a doctor.                                                                        |
+| v1.0    | medical professional                | identify more severe illnesses that patients might have                                                          | be referred to by Doctor Duke so that the patient can get the medical care required.                   |
```

Comment:
Does this fit well to the Target user profile? This feels like an extension to your target use profile, which states it targets users with mild illnesses.